src/test/java/seedu/address/testutil/ClientBuilder.java

```diff
@@ -120,6 +120,14 @@ public ClientBuilder withTimezone(String timezone) {
         return this;
     }
 
+    /**
+     * Sets the {@code Timezone} of the {@code Client} that we are building.
+     */
+    public ClientBuilder withTimezone(Timezone timezone) {
+        this.timezone = timezone;
+        return this;
+    }
+
```

Comment:
If I pass in `Timezone` I can't guarentee what `getCurrHourInTimezone ` returns as the time is always changing. That's why I created `TimezoneStub` so that I can hardcode return value for  `getCurrHourInTimezone ` for testing my predicate